snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/validation/detail/SnomedValidationIssueDetailExtension.java

```diff
@@ -340,11 +368,11 @@ private void extendRelationshipIssueLabels(BranchContext context, Collection<Val
 				
 				final String sourceId = relationshipFragments[0];
 				final String typeId = relationshipFragments[1];
-				final String destinationId = relationshipFragments[2];
+				final String destinationIdOrValue = relationshipFragments[2];
 				
 				final String sourceTerm = affectedComponentLabelsByConcept.getOrDefault(sourceId, sourceId);
 				final String typeTerm = affectedComponentLabelsByConcept.getOrDefault(typeId, typeId);
-				final String destinationTerm = affectedComponentLabelsByConcept.getOrDefault(destinationId, destinationId);
+				final String destinationTerm = affectedComponentLabelsByConcept.getOrDefault(destinationIdOrValue, destinationIdOrValue);
```

Comment:
If a relationship value is equal to an SCTID, and the same SCTID is needed for an actual `destinationId` in some other issue, the label of the concept will be displayed here. I'd suggest using `#` for numbers, and `"` (quotes) for string values above (lines 353 and 355) to avoid this.

Reply:
(Not very likely at the moment, so I have approved the PR nevertheless!)